bleak/__init__.py

```diff
@@ -51,9 +51,9 @@
         BleakClientBlueZDBus as BleakClient,
     )  # noqa
 elif platform.system() == "Darwin":
-    from Foundation import NSClassFromString
-
-    if NSClassFromString("CBPeripheral") is None:
+    try:
+        from CoreBluetooth import CBPeripheral
+    except:
```

Comment:
How about this instead?

```python
except Exception as ex:
    raise BleakError("Bleak requires the CoreBluetooth Framework") from ex
```
Then we don't lose information from the first exception.

Reply:
👍🏻

Reply:
You have got to be kidding me! I have been coding Python for 12 years now and this is the first time I have seen or heard of `raise x from y`!

🤯

I have many times lamented Python's lack of good exception chaining and it has been there all along...

@dlech thank you for this revelation!

Reply:
:grin: